config/locales/views/uploads/actual_histories.en.yml

```diff
@@ -0,0 +1,30 @@
+---
+en:
+  page_title:
+    uploads:
+      actual_histories: Upload actual data history
+  page_content:
+    uploads:
+      actual_histories:
+        actuals_tab:
+          body_html:
+            <p class="govuk-body">Historic actuals can be uploaded into this report.</p>
+            <div class="govuk-inset-text">It is recommended that the actual data history is the only data uploaded in this report.</div>
```

Comment:
Undoubtedly! 😄 

I think I am trying to say:

> don't add your data import to a report that has delivery partner data already added to it.
---
> Historic actuals must be associated with a past financial quarter 

is probably a safe assumption!

I very much see this as a short term fix, once BEIS have all the history, they should no longer use this feature.  I suspect I was trying to keep the amount of work down here by not enforcing this too much, but happy to do so.

Reply:
@ed do you think we should add some enforcing of past quarters on this or shall we merge this and write up a card?

Reply:
I do think we ought to enforce that the actuals being uploaded are indeed historic rather than current. 

I know this is a temp feature, but I'm imagining that it might hang around for a while, get mis-used, cause confusion, burn through support time...

And yes I think the enforcement should follow in a separate PR